documentation/src/main/asciidoc/titles/developing/stories.adoc

```diff
@@ -1,8 +1,100 @@
 include::{stories}/assembly_configuring_maven_repository.adoc[leveloffset=+1]
-include::{stories}/assembly_developing.adoc[leveloffset=+1]
+
+//Cache configuration
+include::{stories}/assembly_cache_modes.adoc[leveloffset=+1]
+include::{stories}/assembly_clustered_caches.adoc[leveloffset=+2]
+include::{stories}/assembly_cache_configuration.adoc[leveloffset=+1]
+include::{stories}/assembly_configuring_statistics_jmx.adoc[leveloffset=+2]
+include::{stories}/assembly_configuring_data_container.adoc[leveloffset=+1]
+include::{stories}/assembly_configuring_persistent_storage.adoc[leveloffset=+1]
+include::{stories}/assembly_creating_custom_cache_stores.adoc[leveloffset=+2]
+include::{stories}/assembly_migrating_cache_stores.adoc[leveloffset=+2]
+include::{stories}/assembly_partition_handling.adoc[leveloffset=+1]
+
+//Transactions and locking
+include::{stories}/assembly_transactions.adoc[leveloffset=+1]
+include::{stories}/assembly_locking.adoc[leveloffset=+1]
+
+//Cross-site replication
+include::{stories}/assembly_xsite_replication.adoc[leveloffset=+1]
+include::{stories}/assembly_xsite_configuring.adoc[leveloffset=+2]
+include::{stories}/assembly_xsite_cli.adoc[leveloffset=+2]
+include::{stories}/assembly_xsite_rest.adoc[leveloffset=+2]
+include::{stories}/assembly_xsite_jmx.adoc[leveloffset=+2]
+include::{stories}/assembly_xsite_logs.adoc[leveloffset=+2]
+
+//Encoding and marshalling
+include::{stories}/assembly_cache_encoding.adoc[leveloffset=+1]
+include::{stories}/assembly_marshalling_protostream.adoc[leveloffset=+1]
+include::{stories}/assembly_marshalling_custom.adoc[leveloffset=+1]
+include::{stories}/assembly_data_conversion.adoc[leveloffset=+1]
+
+//Querying
+include::{stories}/assembly_query_indexing_caches.adoc[leveloffset=+1]
+include::{stories}/assembly_query_creating_ickle_queries.adoc[leveloffset=+1]
+include::{stories}/assembly_query_remote.adoc[leveloffset=+1]
+include::{stories}/assembly_query_embedded.adoc[leveloffset=+1]
+include::{stories}/assembly_query_continuous.adoc[leveloffset=+1]
+include::{stories}/assembly_query_monitoring_tuning.adoc[leveloffset=+1]
+
+//Remote caches
+include::{stories}/assembly_creating_remote_caches.adoc[leveloffset=+1]
+//Java client
+include::{stories}/assembly_hotrod_clients.adoc[leveloffset=+1]
+include::{stories}/assembly_hotrod_client_configuration.adoc[leveloffset=+2]
+include::{stories}/assembly_hotrod_client_api.adoc[leveloffset=+2]
+//REST
+include::{stories}/assembly_rest.adoc[leveloffset=+1]
+include::{stories}/assembly_rest_api.adoc[leveloffset=+2]
+include::{stories}/assembly_rest_client_code.adoc[leveloffset=+2]
+//Embedded caches
+include::{stories}/assembly_creating_embedded_caches.adoc[leveloffset=+1]
+include::{stories}/assembly_cdi.adoc[leveloffset=+1]
+include::{stories}/assembly_clustered_locks.adoc[leveloffset=+1]
+include::{stories}/assembly_cluster_executor.adoc[leveloffset=+1]
+include::{stories}/assembly_streams.adoc[leveloffset=+1]
```

Comment:
it looks like this sections already exist in `Embedding Infinispan in Java Applications`

Reply:
Yes, it appears in the embedded guide as well. This is part of the "hybrid" approach that was suggested in chat. We'll have the "all-in-one" dev guide for searchability alongside the guides that are scoped to more distinct user stories.